libs/blocks/marketo/marketo.js

```diff
@@ -18,13 +18,32 @@ import { parseEncodedConfig, loadScript, createTag } from '../../utils/utils.js'
 const FORM_ID = 'form id';
 const BASE_URL = 'marketo host';
 const MUNCHKIN_ID = 'marketo munckin';
+const FORM_MAP = {
+  'destination url': 'form.success.content',
+  'co-partner names': 'program.copartnernames',
+  'sfdc campaign id': 'program.campaignids.sfdc',
```

Comment:
These look like very odd strings, but I guess that's why this map exist

Reply:
I have created an authoring guide https://milo.adobe.com/docs/authoring/marketo.
We are converting author friendly names to values that are added to `window.mcz_marketoForm_pref`

Reply:
Since those are specific strings and the spaces etc. need to be set, maybe `destination-url` `co-partner-names` and `sfdc-campaign-id` ? 

Not blocking the PR, but it might be more obvious it's something "pasteable" to authors then, but I'll leave it up to you

Reply:
@mokimo That's a good idea, I'll string replace spaces with dashes so we're more flexible with authoring.